bot/cogs/translate_cog.py

```diff
@@ -0,0 +1,179 @@
+import json
+import logging
+import discord
+import aiohttp
+import uuid
+import bot.extensions as ext
+import discord.ext.commands as commands
+from discord.ext.commands.errors import UserInputError
+from bot.bot_secrets import BotSecrets
+from bot.consts import Colors
+
+log = logging.getLogger(__name__)
+
+language_name_to_short_code = {
+    "afrikaans": "af",
+    "arabic": "ar",
+    "bulgarian": "bg",
+    "catalan": "ca",
+    "chinese simplified": "zh-Hans",
+    "chinese traditional":	"zh-Hant",
+    "croatian": "hr",
+    "czech": "cs",
+    "danish": "da",
+    "dutch": "nl",
+    "english": "en",
+    "estonian": "et",
+    "finnish": "fi",
+    "french": "fr",
+    "german": "de",
+    "greek": "el",
+    "gujarati": "gu",
+    "haitian creole": "ht",
+    "hebrew": "he",
+    "hindi": "hi",
+    "hungarian":"hu",
+    "icelandic": "is",
+    "indonesian": "id",
+    "irish": "ga",
+    "italian": "it",
+    "japanese": "ja",
+    "klingon": "tlh-Latn",
+    "korean": "ko",
+    "kurdish (central)": "ku-Arab",
+    "latvian": "lv",
+    "lithuanian": "lt",
+    "malay": "ms",
+    "maltese": "mt",
+    "norwegian": "nb",
+    "pashto": "ps",
+    "persian": "fa",
+    "polish": "pl",
+    "portuguese": "pt",
+    "romanian": "ro",
+    "russian": "ru",
+    "serbian (cyrillic)": "sr-Cyrl",
+    "serbian (latin)": "sr-Latn",
+    "slovak": "sk",
+    "slovenian": "sl",
+    "spanish": "es",
+    "swahili": "sw",
+    "swedish": "sv",
+    "tahitian": "ty",
+    "thai": "th",
+    "turkish": "tr",
+    "ukrainian": "uk",
+    "urdu": "ur",
+    "vietnamese": "vi",
+    "welsh": "cy",
+    "yucatec maya": "yua"
+}
+
+language_short_code_to_name = {value : key for (key, value) in language_name_to_short_code.items()}
```

Comment:
you dont need the () here
```suggestion
language_short_code_to_name = {value : key for key, value in language_name_to_short_code.items()}
```